integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/validation/InServiceRegistryValidator.java

```diff
@@ -0,0 +1,37 @@
+package no.difi.meldingsutveksling.validation;
+
+import lombok.extern.slf4j.Slf4j;
+import no.difi.meldingsutveksling.domain.Organisasjonsnummer;
+import no.difi.meldingsutveksling.serviceregistry.ServiceRegistryLookup;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+@Slf4j
+public class InServiceRegistryValidator implements ConstraintValidator<InServiceRegistry, String> {
+
+    @Autowired
+    private ServiceRegistryLookup serviceRegistryLookup;
+
+    public void initialize(InServiceRegistry constraint) {
+        // NOOP
+    }
+
+    public boolean isValid(String s, ConstraintValidatorContext context) {
+        if (s == null) {
+            return true;
+        }
+
+        try {
+            return serviceRegistryLookup.isInServiceRegistry(getStrippedIdentifier(s));
+        } catch (Exception e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)